ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -1033,6 +1064,14 @@ AtomicInteger getErrorCount() {
     return errorCount;
   }
 
+  /**
+   * Exposed for testing.
+   * @return a boolean value that indicates whether current store is disabled due to I/O error.
+   */
+  AtomicBoolean getDisabledOnError() {
```

Comment:
for testing i would prefer having a setter instead of getter for the AtomicBoolean object.

Reply:
will do